WIP: Add governance.md (#3)
catalyst-network/Community: governance/playbooks/responses/deluge.md

Thanks for reaching out!

Because we treat our issues list as [the team's backlog](https://en.wikipedia.org/wiki/Scrum_(software_development)#Product_backlog), we close issues that are questions since they don't represent a task needing to be completed. For most questions about Electron there are a lot of options.

> Check out the [Electron community](https://github.com/electron/electron#community). There are also a bunch of helpful people at this [community forum](https://discuss.atom.io/c/electron) that should be willing to point you in the right direction.

Reviewer: ```suggestion
Check out the [Catalyst Community](https://github.com/catalyst-network/Community). There are also a bunch of helpful people on [Discord](https://discord.gg/anTP7xm) that should be willing to point you in the right direction.
```
